### PR TITLE
Add sample for show the Health tool usage - Package Gen mode

### PR DESCRIPTION
**Pull request by isuruh15**

### CODE REVIEW

File: working_with_health_tool/main.bal [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+public type Patient record {
+    string id;
+    string lastName?;
+    string firstName;
+    string middleName;
+    string gender?;
+};
+
+service / on httpListener {
```
Comment (nirmal070125) [inline working_with_health_tool/main.bal:15]: Init `httpListener` here itself.

File: working_with_health_tool/main.bal [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    isolated resource function get [string fhirType]/[string id]() returns @http:Payload {mediaType: ["application/fhir+json", "application/fhir+xml"]}r4:FHIRWireFormat|error {
```
Comment (nirmal070125) [inline working_with_health_tool/main.bal:17]: space before `r4:FHIRWireFormat` is missing

File: working_with_health_tool/main.bal [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    ,
+            gender: <carinbb:PatientGender>patient.gender,
+            name: [
+                {
+                    family: patient.lastName,
+                    given: [patient.firstName, patient.middleName]
+                }
+            ]
+        };
+
```
Comment (nirmal070125) [inline working_with_health_tool/main.bal:46]: Can you add a println as this is a sample.

**Review by nirmal070125 — CHANGES_REQUESTED** [review]

File: working_with_health_tool/main.bal (hunk 3 above)
Comment (isuruh15) [reply working_with_health_tool/main.bal:50]: fixed in [82be790](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/82be790415e7ca789008605060f62b95eded8c8a)

File: working_with_health_tool/main.bal (hunk 2 above)
Comment (isuruh15) [reply working_with_health_tool/main.bal]: fixed in [82be790](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/82be790415e7ca789008605060f62b95eded8c8a)

File: working_with_health_tool/main.bal (hunk 1 above)
Comment (isuruh15) [reply working_with_health_tool/main.bal]: fixed in [82be790](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/82be790415e7ca789008605060f62b95eded8c8a)

File: working_with_health_tool/main.bal [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    string middleName;
+    string gender?;
+};
+
+service / on new http:Listener(9090) {
+
+    isolated resource function get [string fhirType]/[string id]() returns @http:Payload {
+        mediaType: [
+            "application/fhir+json",
+            "application/fhir+xml"
```
Comment (nirmal070125) [inline working_with_health_tool/main.bal:19]: remove the fhir+xml

Comment (isuruh15) [reply working_with_health_tool/main.bal]: fixed in [d63567e](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/d63567eabcafd01898407fb964bd003295982e05)

Comment (praneesha) [reply working_with_health_tool/main.bal]: Resolving as per the above comment.

**Pull request merged**